src/test/java/core/basesyntax/FruitShopStorageTests.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitShopStorage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FruitShopStorageTests {
+    private FruitShopStorage fruitShopStorageDefault;
+
+    @BeforeEach
+    void setUp() {
+        fruitShopStorageDefault = new FruitShopStorage();
+        fruitShopStorageDefault.put("banana", 100);
+        fruitShopStorageDefault.put("apple", 100);
+    }
+
+    @Test
+    void fruitShopStorage_createStorage_ok() {
+        FruitShopStorage fruitShopStorage = new FruitShopStorage();
+        Assertions.assertNotNull(fruitShopStorage);
+    }
```

Comment:
Separate test for constructor is usually not required.
You have fruitShopStorageDefault = new FruitShopStorage() on line 14.
So your constructor will be used for every other test and in case it doesn't work you'll have failed other tests
```suggestion
```